frontend/src/app/modules/workspace/personal/personal.component.html

```diff
@@ -1,144 +1,156 @@
+<div>
```

Comment:
It's pretty difficult to grasp the changes with this diff. Can you describe it in two words?

Reply:
I changed the class names of the buttons and moved them to the general styles to be able to style the buttons in the video component in the same way